locale/fi/LC_MESSAGES/django.po

```diff
@@ -3721,6 +3721,10 @@ msgstr "Moduulin suorittamiseen vaaditaan %(points)s pistettä."
 msgid "CHANGES_ARE_POSSIBLE_IN_EXERCISES_BEFORE_MODULE_OPENING"
 msgstr "Tehtäviin saattaa tulla muutoksia ennen kierroksen avautumista!"
 
+#: exercise/templates/exercise/_user_results.html
+msgid "CONVERT_SUBMISSION_BY_MODULE"
+msgstr "Convert Late Submissions in Finnish"
```

Comment:
I can help with Finnish translations, once the terminology is fixed (relating to above comment on English texts)

Reply:
Yes, thanks for the help